app/views/admin/loans/timeline/table/_timeline_table.html.slim

```diff
@@ -0,0 +1,15 @@
+- max_depth = @loan.root_timeline_entry.max_descendant_depth
+
+= render "admin/loans/timeline/table/timeline_header", loan: @loan
```

Comment:
I think the toughest part of this change for me was finding all of the little partials. Some seem to be under loans, and some not. These views could probably get simplifed by removing the hardcoded paths, and allowing rails to find the correct partial. http://api.rubyonrails.org/classes/ActionView/PartialRenderer.html

I think there would be two major benefits to that. First would be that being forced to follow the rails conventions makes it easier for new peeps to find things. The second would make it a little easier to customize a view based on the classes loaded. IE no need to do `is_a?(ProjectGroup)`.

Ideally we would also remove the locals and ivars, but that is a separate discussion. ;)